pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -138,18 +139,19 @@ func EnforceHostNetwork() bool {
 }
 
 func SetRevisionHistoryLimit(data map[string]string) {
-	strval := k8sutil.GetValue(data, revisionHistoryLimitSettingName, revisionHistoryLimitDefaultValue)
-	if strval != "" {
+	strval := k8sutil.GetValue(data, revisionHistoryLimitSettingName, revisionHistoryLimitDefaultStr)
+	var limit int32
+	if strval == "" {
+		limit = revisionHistoryLimitDefault
+	} else {
 		numval, err := strconv.ParseInt(strval, 10, 32)
 		if err != nil {
-			logger.Warningf("failed to parse value %q for %q. assuming default value.", strval, revisionHistoryLimitSettingName)
```

Comment:
Don't we want to tell users when there is a failure setting something they specify?

Reply:
@BlaineEXE  wrote: 
> Don't we want to tell users when there is a failure setting something they specify?

right. That's why I added  these  log messsages originally. But upon request by @travisn , I removed them as part of fixing the build and test runs.

Reply:
Logging the warnings is still good, I was just questioning returning an error that causes the reconcile to fail

Reply:
@travisn  wrote:
> Logging the warnings is still good, I was just questioning returning an error that causes the reconcile to fail

Apologies for "quoting" you wrongly!

I can add back the logging

Reply:
added back the log message.